src/app/public/src/modules/container/container.module.ts

```diff
@@ -0,0 +1,18 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { StacheContainerComponent } from './container.component';
+
+@NgModule({
+  imports: [
+    CommonModule
+  ],
+  declarations: [
+    StacheContainerComponent
+  ],
+  exports: [
+    StacheContainerComponent
+  ],
+  providers: []
+})
+export class StacheContainerModule { }
```

Comment:
Moving `StacheContainerComponent` to a new directory is a breaking change. What is the goal for moving it to its own module?

Reply:
Originally this was moved because the footer was also going to make use of the container, that has changed so it can be moved back.